Prevent java-commons-lang3 from depending on junit-bom when used with
maven.

From 5c15b04ff9d05235460ea58f303fcf3502a943f2 Mon Sep 17 00:00:00 2001
From: "Artyom V. Poptsov" <poptsov.artyom@gmail.com>
Date: Mon, 13 Jun 2022 09:25:42 +0300
Subject: [PATCH] pom.xml: Remove dependency on 'junit-bom'

---
 pom.xml | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/pom.xml b/pom.xml
index 167a85a47..6cd9be57b 100644
--- a/pom.xml
+++ b/pom.xml
@@ -513,18 +513,6 @@
     </contributor>
   </contributors>
 
-  <dependencyManagement>
-    <dependencies>
-      <dependency>
-        <groupId>org.junit</groupId>
-        <artifactId>junit-bom</artifactId>
-        <version>5.7.1</version>
-        <type>pom</type>
-        <scope>import</scope>
-      </dependency>
-    </dependencies>
-  </dependencyManagement>
-
   <!-- Lang should depend on very little -->
   <dependencies>
     <!-- testing -->
-- 
2.25.1

